public/styles.css

```diff
@@ -1,3 +1,61 @@
-body {
-  margin: 0;
+#main {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+.mainBox {
+  flex-direction: column;
+  /*align-items: center;*/
+  justify-content: space-around;
+  display: flex;
+  flex-wrap: wrap;
+  align-self: flex-start;
+}
+.dtborder {
+  position: relative;
+  border: 20px solid #ffecb7;
+  background: #ffffff;
+  padding: 60px;
+  text-align: center;
+  height: 100px;
+  line-height: 100px;
+  width: 100px;
+}
+.dtborder:before {
+  content: "";
+  position: absolute;
+  top: 0px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  border: 20px solid sandybrown;
+}
+
+.dtborder:after {
+  content: "";
+  position: absolute;
+  top: 20px;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+  border: 20px solid #b7ecb7;
+}
```

Comment:
widzę, że pseudoklasami ogarnąłeś sobie kolory paddingów ^^. Sprytnie! Ale chciałbym, żebyś się pomęczył i zrobił tak, żeby 1 element miał zarówno margin, padding i border. Niech będzie to osobny div obok inputów i checkboxa i niech będą one dziećmi box'a. Czyli coś takiego:
```
<div class="box">
  <div class="tutaj element z marginem borderem i paddingiem">Box</div>
  <input width/>
  <input heihgt/>
  <checkbox/>
</div>
```

no i te boxy w mainie tak jak do tej pory. Spróbuj. Wtedy będziesz też widział co daje property box-sizing.